packages/core/examples/buttonGroupExample.tsx

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
same as the other PR, examples have moved to packages/docs-app/src/examples/core-examples/...